### PR TITLE
added gtest

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
any remarks?

File: easybuild/easyconfigs/g/gtest/gtest-1.6.0-intel-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description = "Google's framework for writing C++ tests on a variety of platforms"
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+sources = [SOURCE_ZIP]
+source_urls = ['https://googletest.googlecode.com/files']
+
+with_configure = True
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/g/gtest/gtest-1.6.0-intel-2014b.eb]: one empty line suffices

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/317f82656f3e5e2322f9 for a full test report.

**Comment by JensTimmerman** [comment]
@boegel fixed

**Comment by boegel** [comment]
Will merge as soon Jenkins agrees.

**Comment by boegel** [comment]
going in, thanks @JensTimmerman!

**Pull request merged**